package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@fontoxml/fonto-benchmark-runner",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "description": "This tool can build a fonto-related tool to measure the performance of a function and compare the performances of different functions.",
   "main": "lib/index.js",
   "scripts": {
     "test": "ts-mocha --paths -p test/tsconfig.json \"test/**/*.ts\" --timeout 20000 --extension ts",
-    "build": "tsc; cp src/index.html lib/index.html"
+    "build": "tsc; copyfiles -f \"./src/index.html\" \"./lib\""
```

Comment:
The `;` is linux specific. Please use `&` to run both commands

Reply:
done